packages/module/src/DataView/DataView.tsx

```diff
@@ -1,15 +1,26 @@
 import React from 'react';
-import { Text } from '@patternfly/react-core';
+import { Stack, StackItem } from '@patternfly/react-core';
+import { DataViewPaginationProps } from '../Hooks';
 
-export interface DataViewProps {
-  /** Example prop */
-  text?: string;
+export interface DataViewProps extends DataViewPaginationProps {
+  /** Content rendered inside the data view */
+  children?: React.ReactNode;
+  /** Custom OUIA ID */
+  ouiaId?: string;
 }
 
-export const DataView: React.FunctionComponent<DataViewProps> = ({
-  text = 'This is Data view'
+export const DataView: React.FC<DataViewProps> = ({
+  children, ouiaId = 'DataView'
 }: DataViewProps) => (
-  <Text>{text}</Text>
-);
+  <Stack data-ouia-component-id={`${ouiaId}-stack}`}>
+    {React.Children.map(children, (child, index) => (
+      React.isValidElement(child) ? (
```

Comment:
I think we want to throw an error if the element is valid. There is no point passing things that are not elements and acting as if it was OK.